projects/dsp-ui/src/lib/action/services/sorting.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SortingService {
+
+    constructor() { }
+
+    /**
+     * reverses an array
+     */
+    reverseArray(value: Array<any>): Array<any> {
+        if (value instanceof Array) {
+            return value.slice().reverse();
+        }
+    }
+
+    /**
+     * compares value by value and sorts by alphabetical order
+     * optionally, you can have the array returned to you in reversed order by providing the 'reversed' parameter to the method call
+     */
+    sortByAlphabetical(value: Array<any>, args: string, reversed?: boolean): Array<any> {
```

Comment:
Does array contain objects? Is it `Array<object>`?

Reply:
refactored in fe5dbc8